docs/stats_feature.md

```diff
@@ -0,0 +1,107 @@
+# Performance Statistics Feature
```

Comment:
We won't add a standalone page for this feature. Please rework it and add it as an addition to `advanced_options.md`.

The section could be about the performance statistics in general and a subsection (shorter than this) for how to use it in the CLI.